torchao/quantization/autoquant.py

```diff
@@ -15,20 +15,11 @@
 from .quant_primitives import (
     safe_int_mm,
 )
-from torchao.utils import TORCH_VERSION_AFTER_2_3, TORCH_VERSION_AFTER_2_5
+from torchao.utils import TORCH_VERSION_AFTER_2_3, TORCH_VERSION_AFTER_2_4
 from torchao.quantization.utils import quantize_activation_per_token_absmax
 
 import torch.nn.functional as F
 
-try:
-    from torch._inductor.utils import do_bench
-except ImportError:
-    try:
-        from torch._inductor.runtime.runtime_utils import do_bench
-    except ImportError:
-        from torch._inductor.runtime.benchmarking import benchmarker
-        do_bench = benchmarker.benchmark
-
```

Comment:
do we need to do this?

Reply:
This API has been broken upstream many times in core and AO needs to work with multiple pytorch versions. What's the problem?